### DISCUSSION TITLE
Head-to-Head Duel Matrix

### DISCUSSION THREAD

**Discussion by mavproductions** [akiver/cs-demo-manager#803] [Ideas]
Hey Akiver,
Thanks for providing such a valuable software in an otherwise currently grim stat for statistical analysis for general players of faceit (or those that enjoy inflicting self-pain on premier)

As far as I know, right now there isn't any duel matrix for players in a 5x5 grid showing how often someone killed someone in particular (or died to them)

Is it possible that this feature could be added as a sub-tab on the Main Duels tab?

Again really appreciate your resilience with this software.

**Comment by mavproductions** [comment]
Something like this:

**Leetify Version** 
![image](https://github.com/akiver/cs-demo-manager/assets/6368850/d971b538-6767-4f05-aec9-fb96c7173c7e)

**CSGOStats.gg** 
![image](https://github.com/akiver/cs-demo-manager/assets/6368850/c246f2b8-8c74-4ad3-a2b8-8947ee63b735)

<ins>or perhaps</ins>

**ESEA Version**
![image](https://github.com/akiver/cs-demo-manager/assets/6368850/cf1688eb-744a-4bf5-b694-81bea88f8d3f)

I personally like the compact look of ESEA myself, but having this overhead view of duels would be really nice for sure. :)

**Comment by akiver** [comment]
Available as of v3.5.0

**Reply by mavproductions** [reply]
TYSM!!!!!!! 🤤🤤🥵🥵
